### ISSUE TITLE
Duplicate messages are sent between WhatsApp groups

### ISSUE THREAD

**Issue by fguinez** [42wim/matterbridge#1679]
**Describe the bug**
I have bridged a series of WhatsApp groups, where one of the groups sends only messages and the others only receive. Sometimes a message that was sent hours ago is resent an additional 1 or 2 times for no apparent reason. 

I suspect that it is related to intermittence in my internet service, because I have the same setup in another local network and I have not observed this error. 

**To Reproduce**
1. Configure bridge between multiple WhatsApp groups. Only one group configured as `gateway.in` and everyone else like `gateway.out`
2. Send messages from the 'in' group
3. Eventually, some of the messages will be sent more than once to the 'out' groups. It seems to coincide with times when the internet is intermittently.

**Expected behavior**
Messages are sent only once.

**Screenshots/debug logs**
The error occurs unpredictably, as soon as I have logs of `matterbridge -debug` I will upload them to this issue.

**Environment:**
 - OS: Raspbian GNU/Linux 10 (buster)
 - Matterbridge version: 1.23.2-linux-armv6

**Additional context**
 Snippet of config:
```
[whatsapp.mywhatsapp]
Number="+50123456789"

SessionFile="session-50123456789.gob"
QrOnWhiteTerminal=false

RemoteNickFormat=""
Buffer=50

[[gateway]]
name="gateway1"
enable=true

[[gateway.in]]
account="whatsapp.mywhatsapp"
channel="123456789123456789@g.us"

# 1
[[gateway.out]]
account="whatsapp.mywhatsapp"
channel="12345678912-3456789123@g.us"

# 2
[[gateway.out]]
account="whatsapp.mywhatsapp"
channel="123456789123456789@g.us"

...

# 35
[[gateway.out]]
account="whatsapp.mywhatsapp"
channel="12345678912-3456789123@g.us"

# 36
[[gateway.out]]
account="whatsapp.mywhatsapp"
channel="123456789123456789@g.us"
```
This is an adaptation of the configuration file, the id of the WhatsApp groups were replaced by dummy data and all groups with configuration [[gateway.out]] from 3 to 34 remain implicit.

**Comment by juliofz** [comment]
@fguinez check out my comment on this related issue #1708. My solution fixed this problem, it seems.

**Comment by fguinez** [comment]
After a few weeks testing what @juliofz  proposed, I think that the problem is effectively solved in #1708.

Thanks!
